### PR TITLE
Exec copy july 27 2022

### PR DESCRIPTION
**Pull request by prose11**
@patrick-j-govalpha any formatting suggestions particularly welcomed :)

### CODE REVIEW

File: governance/votes/Executive vote - July 27, 2022.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+title: Template - [Executive Vote] Real World Assets Vault Onboardings - July 27, 2022
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - July 27, 2022.md:2]: Not sure if "Real World Asset Vault Onboardings" is better
Or even "Onboarding Real World Asset Vaults"

File: governance/votes/Executive vote - July 27, 2022.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- Societe Generale – Forge (OFH) will be onboarded as a new vault type, RWA-008, as detailed below. 
+- Huntingdon Valley Bank (HVBank) will be onboarded as a new vault type, RWA-009, as detailed below. 
+
+
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - July 27, 2022.md:22]: Delete extra line

File: governance/votes/Executive vote - July 27, 2022.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+As per [this poll](https://vote.makerdao.com/polling/QmajCtnG#vote-breakdown), RWA-008 Societe Generale – Forge (OFH) will be onboarded with the following parameters, if this executive proposal passes.
+
+#### Collateral Parameters
+
+* [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee): 0.05%
+* [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling): 30 million DAI
+* [Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio): 100%
+* Liquidation Process: See [MIP21](https://mips.makerdao.com/mips/details/MIP21) for details.
+
+#### Relevant Smart-Contract Information
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - July 27, 2022.md:46]:
```suggestion
#### Relevant Smart Contract Information
```
Remove hyphen

File: governance/votes/Executive vote - July 27, 2022.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+"NAME": "RWA-008"  
+"ILK": "RWA008-A"  
+"RWA008": "0x7907FB7f6aa0ebe363BC6966c4132849b7B560F8"  
+"MCD_JOIN_RWA008_A": "0x8EeE937787DFD6A82ED1F1d11d9db7e2dC3a5884"  
+"RWA008_A_URN": "0x7b7061164858B69cFc3B0a3B8b630C9279c20de5"  
+"RWA008_A_URN_CLOSE_HELPER": "0xDCd1D2879A309785b4E40b6Fb3FbE36D315bC597"  
+"RWA008_A_INPUT_CONDUIT": "0xC5b09E985B396C1D3883030404DDE06aef71e972"  
+"RWA008_A_OUTPUT_CONDUIT": "0x2d2962fbDF6d5d9567387e33901BF83866431e09"  
+"RWA008_A_OPERATOR": "0x03f1A14A5b31e2f1751b6db368451dFCEA5A0439"  
+"RWA008_A_MATE": "0xb9444802F0831A3EB9f90E24EFe5FfA20138d684"
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - July 27, 2022.md:60]: Change formatting as discussed

```suggestion
* **MIP21_LIQUIDATION_ORACLE:** `0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30`. 
* **RWA_TOKEN_FAB**: `0x16c299c3fB950Cdea730cA6C456aDcE44CbeD350`.
* **SYMBOL:** `RWA008`.
* **NAME:** `RWA-008`.
* **ILK:** `RWA008-A`.
* **RWA008:** `0x7907FB7f6aa0ebe363BC6966c4132849b7B560F8`.
* **MCD_JOIN_RWA008_A**: `0x8EeE937787DFD6A82ED1F1d11d9db7e2dC3a5884`.
* **RWA008_A_URN:** `0x7b7061164858B69cFc3B0a3B8b630C9279c20de5`.
* **RWA008_A_URN_CLOSE_HELPER** `0xDCd1D2879A309785b4E40b6Fb3FbE36D315bC597`.
* **RWA008_A_INPUT_CONDUIT:** `0xC5b09E985B396C1D3883030404DDE06aef71e972`.
* **RWA008_A_OUTPUT_CONDUIT:** `0x2d2962fbDF6d5d9567387e33901BF83866431e09`.
* **RWA008_A_OPERATOR:** `0x03f1A14A5b31e2f1751b6db368451dFCEA5A0439`.
* **RWA008_A_MATE:** `0xb9444802F0831A3EB9f90E24EFe5FfA20138d684`.
```

File: governance/votes/Executive vote - July 27, 2022.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+"MIP21_LIQUIDATION_ORACLE": "0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30"  
+"RWA_TOKEN_FAB": "0x16c299c3fB950Cdea730cA6C456aDcE44CbeD350"  
+"SYMBOL": "RWA009"  
+"NAME": "RWA-009"  
+"ILK": "RWA009-A"  
+"RWA009": "0x518351716C4Ec0531B426b174ed79FfBcc9A4242"  
+"MCD_JOIN_RWA009_A": "0xAe20C8f0e9dc566bDf5E5628fc2EA0E66fd60437"  
+"RWA009_A_URN": "0xDE70058B9C22012e7E4c53aC3F117864cE8C59CA"  
+"RWA009_A_JAR": "0xB67c844C8b2B78125978dEcf33A48d0175486322"  
+"RWA009_A_OUTPUT_CONDUIT": "0x508D982e13263Fc8e1b5A4E6bf59b335202e36b4"
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - July 27, 2022.md:98]: Change formatting as discussed

```suggestion
* **MIP21_LIQUIDATION_ORACLE:** `0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30`.
* **RWA_TOKEN_FAB:** `0x16c299c3fB950Cdea730cA6C456aDcE44CbeD350`.
* **SYMBOL:** `RWA009`.
* **NAME:** `RWA-009`.
* **ILK:** `RWA009-A`.
* **RWA009:** `0x518351716C4Ec0531B426b174ed79FfBcc9A4242`.
* **MCD_JOIN_RWA009_A:** `0xAe20C8f0e9dc566bDf5E5628fc2EA0E66fd60437`.
* **RWA009_A_URN:** `0xDE70058B9C22012e7E4c53aC3F117864cE8C59CA`.
* **RWA009_A_JAR:** `0xB67c844C8b2B78125978dEcf33A48d0175486322`.
* **RWA009_A_OUTPUT_CONDUIT:** `0x508D982e13263Fc8e1b5A4E6bf59b335202e36b4`.
```

File: governance/votes/Executive vote - July 27, 2022.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+As per [this poll](https://vote.makerdao.com/polling/QmQMDasC#vote-breakdown), RWA-009 Huntingdon Valley Bank (HVBank) will be onboarded with the following parameters, if this executive proposal passes.
+ 
+#### Collateral Parameters
+
+* [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee): 0.0%
+* [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling): 100 million DAI
+* [Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio): 100%
+* Liquidation Process: See [MIP21](https://forum.makerdao.com/t/mip21-real-world-assets-off-chain-asset-backed-lender/3917) for details.
+
+#### Relevant Smart-Contract Information
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - July 27, 2022.md:87]: Remove hyphen
```suggestion
#### Relevant Smart Contract Information
```

File: governance/votes/Executive vote - July 27, 2022.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+### RWA-008 Societe Generale – Forge (OFH) Onboarding
+
+As per [this poll](https://vote.makerdao.com/polling/QmajCtnG#vote-breakdown), RWA-008 Societe Generale – Forge (OFH) will be onboarded with the following parameters, if this executive proposal passes.
+
+#### Collateral Parameters
+
+* [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee): 0.05%
+* [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling): 30 million DAI
+* [Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio): 100%
+* Liquidation Process: See [MIP21](https://mips.makerdao.com/mips/details/MIP21) for details.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - July 27, 2022.md:44]: SocGen links to the MIPs portal and this one links to the forum, probs best they link to the same source

File: governance/votes/Executive vote - July 27, 2022.md (hunk 1 above)
Comment (prose11) [reply governance/votes/Executive vote - July 27, 2022.md:2]: Much better :)

File: governance/votes/Executive vote - July 29, 2022.md [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- Amendments to the letter agreements between RWA Foundation and various parties for the RWA-009 (HVBank) legal construction.
```
Comment (miguel567) [inline governance/votes/Executive vote - July 29, 2022.md:19]: Can we add a link to this forum post from Christian with a bit more context? https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/8?u=miguel_cabeza

File: governance/votes/Executive vote - July 29, 2022.md [hunk 9]
```diff
@@ -0,0 +1,199 @@
...
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### RWA-008 Societe Generale – Forge (OFH) Onboarding
+
+As per [this poll](https://vote.makerdao.com/polling/QmajCtnG#vote-breakdown), RWA-008 Societe Generale – Forge (OFH) will be onboarded with the following parameters, if this executive proposal passes.
```
Comment (miguel567) [inline governance/votes/Executive vote - July 29, 2022.md:39]: Can you remove (OFH) from the wording. We are actually not onboarding the token in the maker protocol due to the token features, so adding it here might be misleading. The Technical Assessment has context on the implementation.

File: governance/votes/Executive vote - July 29, 2022.md [hunk 10]
```diff
@@ -0,0 +1,199 @@
...
+* **RWA008_A_INPUT_CONDUIT:** `0xa397a23dDA051186F202C67148c90683c413383C`.
+* **RWA008_A_OUTPUT_CONDUIT:** `0x21CF5Ad1311788D762f9035829f81B9f54610F0C`.
+* **RWA008_A_OPERATOR:** `0x03f1A14A5b31e2f1751b6db368451dFCEA5A0439`.
+* **RWA008_A_MATE:** `0xb9444802F0831A3EB9f90E24EFe5FfA20138d684`.
+
+*Please note that **RWA008_A_OPERATOR** represents Societe Generale – Forge and **RWA008_A_MATE** represents DIIS group.*
+
+#### Additional Information
+
+Please review the following forum threads containing information about RWA-008 (OFH) to inform your position before voting.
```
Comment (miguel567) [inline governance/votes/Executive vote - July 29, 2022.md:69]: Sames as above, I would remove the mention to (OFH)

File: governance/votes/Executive vote - July 29, 2022.md [hunk 11]
```diff
@@ -0,0 +1,199 @@
...
+* **NAME:** `RWA-009`.
+* **ILK:** `RWA009-A`.
+* **RWA009:** `0x8b9734bbaA628bFC0c9f323ba08Ed184e5b88Da2`.
+* **MCD_JOIN_RWA009_A:** `0xEe0FC514280f09083a32AE906cCbD2FAc4c680FA`.
+* **RWA009_A_URN:** `0x1818EE501cd28e01E058E7C283E178E9e04a1e79`.
+* **RWA009_A_JAR:** `0x6C6d4Be2223B5d202263515351034861dD9aFdb6`.
+* **RWA009_A_OUTPUT_CONDUIT:** `0x508D982e13263Fc8e1b5A4E6bf59b335202e36b4`.
+
+*Please note that:*
+- **RWA009_A_OUTPUT_CONDUIT** represents Genesis trading address (receivers of DAI who will swap to fiat).
```
Comment (miguel567) [inline governance/votes/Executive vote - July 29, 2022.md:106]: Can you add that the address can be verified here: https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/10?u=miguel_cabeza

File: governance/votes/Executive vote - July 29, 2022.md (hunk 9 above)
Comment (prose11) [reply governance/votes/Executive vote - July 29, 2022.md:40]: Hm I guess that makes sense, is a little confusing for the community considering this was the approval poll tho: 

https://vote.makerdao.com/polling/QmajCtnG#vote-breakdown

Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - July 29, 2022.md:40]: To add, the wording of that poll was written by Will from RWF so we went with what he thought was appropriate

File: governance/votes/Executive vote - July 29, 2022.md [hunk 12]
```diff
@@ -0,0 +1,204 @@
...
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- [Amendments to the letter agreements](https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/8) between RWA Foundation and various parties for the RWA-009 (HVBank) legal construction. 
+- 25 million DAI will be drawn down in the RWA-009 (HVBank) vault.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - July 29, 2022.md:20]: Extremely minor, so we might want to move changes beneath additions as we are adding the HVBank vault and then drawing down the 25 million, so it's a little backwards right now

**Review by patrick-j-govalpha — COMMENTED** [review]
@prose11 one minor comment but otherwise I think we're g2g. Formatting the legalese is definitely tricky

File: governance/votes/Executive vote - July 29, 2022.md (hunk 12 above)
Comment (prose11) [reply governance/votes/Executive vote - July 29, 2022.md:20]: Honestly was thinking about this too, maybe just swapping additions and changes?

File: governance/votes/Executive vote - July 29, 2022.md [hunk 13]
```diff
@@ -0,0 +1,204 @@
...
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- Societe Generale – Forge (OFH) will be onboarded as a new vault type, RWA-008, as detailed below. 
+- Huntingdon Valley Bank (HVBank) will be onboarded as a new vault type, RWA-009, as detailed below.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - July 29, 2022.md:24]: So change it to this?
```suggestion
If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
- Societe Generale – Forge (OFH) will be onboarded as a new vault type, RWA-008, as detailed below. 
- Huntingdon Valley Bank (HVBank) will be onboarded as a new vault type, RWA-009, as detailed below. 


If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
- [Amendments to the letter agreements](https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/8) between RWA Foundation and various parties for the RWA-009 (HVBank) legal construction. 
- 25 million DAI will be drawn down in the RWA-009 (HVBank) vault.
```

File: governance/votes/Executive vote - July 29, 2022.md (hunk 12 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - July 29, 2022.md:20]: Yeah I think so - see: https://github.com/makerdao/community/pull/1877/files#r932480224

**Review by patrick-j-govalpha — APPROVED** [review]
LGTM!

**Pull request merged**